docs/platforms/elixir/integrations/oban/index.mdx

```diff
@@ -9,7 +9,11 @@ The [Oban](https://getoban.pro) integration supports:
   * Capturing errors and exceptions that happen in Oban jobs.
   * Cron monitoring for scheduled Oban jobs.
 
-This integration is built into the Sentry SDK.
+This integration is built into the Sentry SDK since *v10.2.0*, and supports automatic monitoring since *v10.3.0*. It requires:
+
+  1. Oban to be a dependency of your application.
+  1. Oban version 2.17.6 or greater.
+  1. Elixir 1.13 or later, since that is required by Oban itself.
```

Comment:
```suggestion

  1. Oban version 2.17.6 and up to be a dependency of your application.
  1. Elixir version 1.13 and up, (since that's required by Oban).
```